whois-api/src/main/java/net/ripe/db/whois/api/httpserver/IpBlockListFilter.java

```diff
@@ -37,7 +36,7 @@ public void init(FilterConfig filterConfig) throws ServletException {}
     public void doFilter(final ServletRequest servletRequest, final ServletResponse servletResponse, final FilterChain filterChain) throws IOException, ServletException {
         final HttpServletRequest httpRequest = (HttpServletRequest) servletRequest;
 
-        if (isBlockedIp(httpRequest.getRemoteAddr())){
+        if (ipBlockManager.isBlockedIp(InetAddresses.forString(httpRequest.getRemoteAddr()))){
```

Comment:
Add an overloaded method `isBlockedIp` to accommodate `request.getRemoteAddress()` so you can handle the conversion within `IpBlockManager`